run_dir/static/js/external/textarea-caret.3.1.0.js

```diff
@@ -0,0 +1,138 @@
+/* jshint browser: true */
```

Comment:
Is this script unmodified from the original (since its placed under external) ? If it hasn't been modified, could you also add a link to the source, so that we could find it to update it later if need be.

Reply:
Fixed!